python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -979,6 +1001,36 @@ def __init__( self, settingsNode, **kw ) :
 
 RenderPassEditor._Spacer = _Spacer
 
+## \todo Should this be a new displayMode of BoolPlugValueWidget?
+class _ToggleGroupingPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plugs, **kw ) :
+
+		self.__row = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 )
+
+		GafferUI.PlugValueWidget.__init__( self, self.__row, plugs )
+
+		self.__groupingModeButton = GafferUI.Button( image = "pathListingList.png", hasFrame=False )
+		self.__groupingModeButton.clickedSignal().connect( Gaffer.WeakMethod( self.__groupingModeButtonClicked ), scoped = False )
+		self.__row.append(
+			self.__groupingModeButton
+		)
+
+	def __groupingModeButtonClicked( self, button ) :
+
+		[ plug.setValue( not plug.getValue() ) for plug in self.getPlugs() ]
+
+	@staticmethod
+	def _valuesForUpdate( plugs, auxiliaryPlugs ) :
+
+		return [ p.getValue() for p in plugs ]
```

Comment:
I think we can omit this and just rely on the base class implementation.

Reply:
Nice! Removed in f153628.